<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3263827</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/kompactor.png</normaloff>:/kompactor.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" colspan="2">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="movieLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="LogWidget" name="reportWidget">
          <property name="contextMenuPolicy">
           <enum>Qt::ActionsContextMenu</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="pathGroupBox">
      <property name="title">
       <string>Path</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="labelAlignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="inputPushButton">
         <property name="text">
          <string>Input</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="PathLineEdit" name="inputPathLineEdit">
         <property name="placeholderText">
          <string>Input folder poath</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="outputPushButton">
         <property name="text">
          <string>Output</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="PathLineEdit" name="outputPathLineEdit">
         <property name="placeholderText">
          <string>Output folder path</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="settingsGroupBox">
      <property name="title">
       <string>Settings</string>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="vertexPrecisionLabel">
         <property name="text">
          <string>Vertex Precision</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="vertexPrecisionComboBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="normalPrecisionLabel">
         <property name="text">
          <string>Normal Precision</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="normalPrecisionComboBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="ProgressBarButton" name="progressBar">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="format">
       <string>Execute</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PathLineEdit</class>
   <extends>QLineEdit</extends>
   <header>PathLineEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>ProgressBarButton</class>
   <extends>QProgressBar</extends>
   <header>ProgressBarButton.hpp</header>
  </customwidget>
  <customwidget>
   <class>LogWidget</class>
   <extends>QTableView</extends>
   <header>LogWidget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
